pkg/apis/config/types.go

```diff
@@ -58,3 +60,11 @@ type ETCDBackup struct {
 	// Schedule is the etcd backup schedule.
 	Schedule *string
 }
+
+// BastionConfig is the config for the Bastion
+type BastionConfig struct {
+	// ImageRef is the openstack image reference
+	ImageRef *string
+	// FlavorRef is the openstack flavorRef reference
+	FlavorRef *string
```

Comment:
now that they are not optional anymore, we do not need the pointer
```go
	ImageRef string
	// FlavorRef is the openstack flavorRef reference
	FlavorRef string
```

Reply:
Thanks, committed in abcddfa9